### ISSUE TITLE
Regression: High Contrast theme: style of unfocused tab dissappeared after updating TreeStyleTab

### ISSUE THREAD

**Issue by actionless** [piroor/treestyletab#2210]
## Steps to reproduce

 1. Start Firefox with clean profile.
 2. Install TST.
 3. Open more than 1 tab
4.1 Observe unfocused tab
4.2 Hover mouse cursor on unfocused tab

## Expected result
1) Tab should have window colors.
2) Background should change on hover.

## Actual result
1) Tab have background color as an Input field.
2) Background is not changed when hovering a tab.

## Environment

 * Platform (OS): Linux
 * Version of Firefox: 66.0.2
 * Version (or revision) of Tree Style Tab: 3.0.3

**Comment by stapuft** [comment]
its happening on all themes, my hover over action is gone aswell, i still get the tooltips, if the tabs name is really long, or its a collapsed parent tab, but no changing color of the tab

**Comment by piroor** [comment]
@actionless Thanks, this regression should be fixed with 836869d88df156ba682a0b4c29f4e5547c3926e9.

@stapuft Did you talk about other themes? I couldn't reproduce any problem around "hover" style on "Plain", "Plain Dark", "Mixed", "Vertigo" and "Sidebar". "Metal" doesn't provide special appearance for hover tabs.

**Comment by stapuft** [comment]
well i guess i am talking about another theme then, or it could be a conflict with my custom css, i have been messing with it to get things looking just right, but i am using a customized version of "plain dark" would you happen to know what its values for the hover over action should be so i can check with my css to discover the conflict?

**Comment by piroor** [comment]
@stapuft The combination of the built-in CSS rules and your custom style rules may conflict, so updating of custom style rules is strongly recommended for every changes around TST's built-in CSS rules. (By the way, I didn't do any change around hover tabs on the "Plain Dark" so I cannot believe that my changes break appearance of hover tabs...)

**Comment by szeder** [comment]
> @actionless Thanks, this regression should be fixed with [836869d](https://github.com/piroor/treestyletab/commit/836869d88df156ba682a0b4c29f4e5547c3926e9).

Thanks for the quick fix.

**Comment by actionless** [comment]
@piroor thanks a lot!
i've built webextension locally and tried in dev-edition - High Contrast theme got back to normal

**Comment by stapuft** [comment]
> 
> 
> @stapuft I didn't do any change around hover tabs on the "Plain Dark" so I cannot believe that my changes break appearance of hover tabs...)

@piroor  i wasnt trying to say you did, i was trying to say that i dont really know what i did to make it go away, and if i knew what the original code that made the tabs have a hover action, i could check my custom code, to see if anything i put in, overwrote the default action.

also as of now the only hover action my tabs do is the tooltip, i had also given them the tab numbering code, but that stopped working along with the other hover over actions.


here is the code i have in there now, i wish i knew what i did wrong.

```
.tab {
background-color: var(--colorE);
height: var(--tab-height);
border-top: none;
border-right: none;
border-left: none;
border-bottom: 1px solid var(--colorF);
}

.tab {
height: 19px;
}

.label {
font-size: 11px;
}

/* Show title of unread tabs with red and italic font */
.tab.unread .label {
  color: blue !important;
  font-style: italic !important;
  font-weight: bold;
}
*/

/* Add private browsing indicator per tab */
.tab.private-browsing .label:before {
  content: "🕶";
}
*/

*//* Change styling of pending (unloaded) tabs */
.tab.discarded {
  opacity: 0.75;
}

.tab.discarded .label-content {
  color: red;
}

.tab.active {
  height: 20px !important;
  background-color: #193B99;
}
.tab.active .label {
  font-weight: bold;
  font-size: 14px;
}
.tab.active .twisty,
.tab.active .label,
.tab.active .counter {
  color: #fff;
}

*/*/All of the options in this section need the raw count of tabs available. This is done using CSS counters:*/
#tabbar {
  counter-reset: vtabs atabs tabs;
  /* vtabs tracks visible tabs, atabs tracks active tabs, tabs tracks all tabs */
}
.tab:not(.collapsed):not(.discarded) {
  counter-increment: vtabs atabs tabs;
}
.tab:not(.collapsed) {
  counter-increment: vtabs tabs;
}
.tab:not(.discarded) {
  counter-increment: atabs tabs;
}
.tab {
  counter-increment: tabs;
}
.tab .burster::after {
  background: Highlight;
  color: HighlightText;
  content: counter(vtabs);
  font-size: x-small;
  right: 0.2em;
  padding: 0.2em;
  pointer-events: none;
  position: absolute;
  bottom: 0.2em;
}
.tab .burster::after {
  opacity: 0;
  transition: 0.2s;
}
.tab:hover .burster::after {
  opacity: 1;
}

*/*/Both of the options in this section use the following block of CSS to position some text in the New Tab button. The text that is displayed is specified in each option below.*/

.newtab-button::after {
  content: var(--tab-count-text);
  pointer-events: none;
    
  position: absolute;
  left: 0.5em;

  /* TST 2.4.0 - Fix for Issue #1664 */
  background: transparent !important;
  mask: none !important;
}
.newtab-button {
  --tab-count-text: counter(tabs) " tabs";
}
.newtab-button {
  --tab-count-text: counter(atabs) "/" counter(vtabs) "/" counter(tabs) " tabs"; 
}
.newtab-button-box {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  height: 20px;
}
#tabbar {
  top: 20px !important;
  bottom: 0 !important;
}
.tab.pinned {
  margin-top: 20px;
}

*/


```
